scripts/scil_analyse_lesions_load.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+This script will output informations about lesions load of a bundle.
+Either using as streamlines, binary map or a bundle voxel labels map.
```

Comment:
The input can either be streamlines, binary bundle map, or a bundle voxel label map.